packages/button/index.js

```diff
@@ -26,7 +26,7 @@ class WarpButton extends kebabCaseAttributes(LitElement) {
     href: { type: String, reflect: true },
     target: { type: String, reflect: true },
     rel: { type: String, reflect: true },
-    buttonClass: { type: String, reflect: true },
+    fullwidth : { type: Boolean, reflect: true },
```

Comment:
I see that in other web components in this package two-word properties would be written in camelCase, and then linked to corresponding attributes in kebab-case (see a `kebabCaseAttributes` wrapper in line 16 or [/packages/epandable/index.js](https://github.com/warp-ds/elements/blob/next/packages/expandable/index.js#L10-L23) for more examples).

Reply:
Nice catch! I will change the prop to "fullWidth" instead. :)